spring-integration-azure/spring-integration-azure-test/src/main/java/com/microsoft/azure/spring/integration/test/support/MessageHandlerTest.java

```diff
@@ -33,6 +33,7 @@
 
     protected DefaultMessageHandler handler;
     protected String destination = "dest";
+    protected String dynamicDestination = "dynamicName";
```

Comment:
not sure if `dynamicDestination ` can be final or not.

Reply:
could be final.